pinot-core/src/test/java/org/apache/pinot/server/realtime/ControllerLeaderLocatorTest.java

```diff
@@ -68,45 +69,48 @@ public void testInvalidateCachedControllerLeader() {
 
     // Create Controller Leader Locator
     FakeControllerLeaderLocator.create(helixManager);
-    ControllerLeaderLocator controllerLeaderLocator = FakeControllerLeaderLocator.getInstance();
+    FakeControllerLeaderLocator controllerLeaderLocator = FakeControllerLeaderLocator.getInstance();
 
     // check values at startup
-    Assert.assertTrue(controllerLeaderLocator.isCachedControllerLeaderInvalid());
+    Assert.assertFalse(controllerLeaderLocator.isCachedControllerLeaderValid());
     Assert.assertEquals(controllerLeaderLocator.getLastCacheInvalidateMillis(), 0);
 
     // very first invalidate
+    long currentTimeMs = System.currentTimeMillis();
```

Comment:
Use fixed values for time. e.g. currentTimeMs = 100, and  go from there.

Reply:
`System.currentTimeMillis();` has been removed from the test.